components/org.wso2.carbon.dashboards.core/src/main/java/org/wso2/carbon/dashboards/core/internal/dao/DashboardMetadataDAO.java

```diff
@@ -30,46 +30,6 @@
 public interface DashboardMetadataDAO {
 
     /**
-     * Check whether a DashboardMetadata exists with given url
-     *
-     * @param url URL of the DashboardMetadata instance
-     * @return true if a DashboardMetadata exists  with given information, false otherwise.
-     * @throws DashboardException if an error occurs
-     */
-    boolean isExists(String url) throws DashboardException;
```

Comment:
why are these methods removed from the interface? removing APIs is not correct because external people are using it.